components/rhs_header_post/rhs_header_post.jsx

```diff
@@ -145,13 +163,21 @@ export default class RhsHeaderPost extends React.Component {
                     {back}
                     <FormattedMessage
                         id='rhs_header.details'
-                        defaultMessage='Message Details'
+                        defaultMessage='Thread'
                     />
+                    {channelName &&
+                        <button
+                            onClick={this.handleJumpClick}
+                            className='style--none sidebar--right__title__channel'
+                        >
+                            {channelName}
```

Comment:
Is it intentional that this label will change from "Direct Message" to the other user's name in a DM channel? If so, it's not consistent with the label that appears on search results in this same channel
![image](https://user-images.githubusercontent.com/3277310/80377285-11cfdc00-8869-11ea-84e0-abf1c3574b2e.png)
![image](https://user-images.githubusercontent.com/3277310/80377267-0bd9fb00-8869-11ea-9925-3d5963373224.png)
If you did want to change the channel name displayed in the search result header, that comes from the `SidebarRight` component.

Reply:
I think we can make it consistent and have the user's name appear.
So for this ticket, as its touching only the sidebar, I think we can have the user's name in the header, similar to the channel names, and create a new ticket to change it for other search related items.

![Screenshot 2020-04-27 at 6 41 40 PM](https://user-images.githubusercontent.com/11034289/80378899-c7754c00-88b6-11ea-8b0e-3880d728185c.png)

Reply:
@esethna ^

Reply:
I'll update the Pinned posts thing with the DM/GM name instead of the generic Direct Messages for now to be consistent.